editor/filesystem_dock.cpp

```diff
@@ -1562,22 +1562,22 @@ void FileSystemDock::_update_resource_paths_after_move(const HashMap<String, Str
 	}
 }
 
-void FileSystemDock::_update_dependencies_after_move(const HashMap<String, String> &p_renames, const Vector<String> &p_remaps) const {
+void FileSystemDock::_update_dependencies_after_move(const HashMap<String, String> &p_renames, const Vector<String> &p_file_owners) const {
 	// The following code assumes that the following holds:
 	// 1) EditorFileSystem contains the old paths/folder structure from before the rename/move.
 	// 2) ResourceLoader can use the new paths without needing to call rescan.
-	List<String> scenes_to_reload;
-	for (int i = 0; i < p_remaps.size(); ++i) {
+	Vector<String> scenes_to_reload;
```

Comment:
Why did you change it to Vector? A list is better in this case.

Reply:
In the "Note" below [Disallowed features](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#disallowed-features), `Vector` seems to be more recommended. I have always used `List` as a queue.

Reply:
I'd expect list to perform better in case where you only append and iterate things. You don't know its size beforehand and don't need random access. But maybe Vector is implemented in a way that makes it more efficient here, idk. It doesn't really matter anyway, unless you have a thousand opened scenes.

Reply:
Do I need to change it back?

Reply:
You can leave it. Maybe someone more knowledgeable will give feedback.